outline/setup_win7.md

```diff
@@ -56,7 +56,7 @@ Go to the [Light Table site](http://www.lighttable.com/). On the page there, you
 ![Light Table downloads](img/light-table-download.png)
 ![Light Table downloads Windows](img/win/light-table-download.png)
 
-Unzip this file (either by finding it in your Downloads folder and double-clicking it, or by choosing "Open" when downloading.) Inside the .zip file, there is a a directory called "Light Table". Drag this to your desktop. (If you know what you are doing and want this somewhere else, that is fine.)
+Unzip this file (either by finding it in your Downloads folder and double-clicking it, or by choosing "Open" when downloading.) Inside the .zip file, there is a a directory called "Light Table". Drag this to your desktop. (If you know what you are doing and want this somewhere else, that is fine; however, the location you choose should not have spaces anywhere in the file path.)
```

Comment:
How about removing _that is fine_ and _however_?

_(If you know what you are doing and want this somewhere else, the location you choose should not have spaces anywhere in the file path.)_